Doc/library/csv.rst

```diff
@@ -408,9 +411,12 @@ Dialects support the following attributes:
 
 .. attribute:: Dialect.escapechar
 
-   A one-character string used by the writer to escape the *delimiter* if *quoting*
-   is set to :const:`QUOTE_NONE` and the *quotechar* if *doublequote* is
-   :const:`False`. On reading, the *escapechar* removes any special meaning from
+   A one-character string used by the writer to escape the *delimiter*,
+   the *quotechar*, ``'\r'``, ``'\n'`` and any of the characters in
+   *lineterminator* if *quoting* is set to :const:`QUOTE_NONE`,
+   the *quotechar* if *doublequote* is :const:`False`,
+   and the *escapechar* itself.
```

Comment:
What about this?
```suggestion
   A one-character string used by the writer to escape characters that
   require escaping:

      * the *delimiter*, the *quotechar*, ``'\r'``, ``'\n'`` and any of the
        characters in *lineterminator* are escaped if *quoting* is set to
        :const:`QUOTE_NONE`;
      * the *quotechar* is escaped if *doublequote* is :const:`False`;
      * the *escapechar* itself.

```

Reply:
ah sorry for the noise above, this looks much better to me.